src/main/java/com/zombiecastlerush/building/Combat.java

```diff
@@ -0,0 +1,62 @@
+package com.zombiecastlerush.building;
+
+
+import com.zombiecastlerush.entity.Role;
+import com.zombiecastlerush.util.Parser;
+import com.zombiecastlerush.util.Prompter;
+
+import java.util.List;
+import java.util.Random;
+
+public class Combat extends Challenge {
+    public Combat(String description) {
+        super(description);
+    }
+
+    public static void combat(Role player, Role enemy) {
+        String msg = "Welcome to Combat mode, what would you like to do?";
+        String combatChoice = Prompter.getUserInput(msg);
+        while (player.getHealth() > 0 && enemy.getHealth() > 0) {
+            System.out.println("fight" +
+                    "run-away");
+            if (combatChoice.isEmpty()) {
+                continue;
+            }
+            combatChoiceParser(combatChoice, player, enemy);
+        }
+    }
+
+    private static void combatChoiceParser(String choice, Role player, Role enemy) {
```

Comment:
throwaway code, but you already knew that.

Reply:
Deleted Code